app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,101 @@
+    <string name="night">Nacht</string>
+    <string name="no_internet_content">Internetzugriff muss vorhanden sein, um Wetterdaten anzuzeigen</string>
+    <string name="no_internet_mobile_data">Mobile Daten</string>
+    <string name="no_internet_title">Kein Internetzugriff</string>
+    <string name="no_internet_wifi">WiFi</string>
+    <string name="north">Wind aus Süden</string>
+    <string name="north_east">Wind in nordöstlicher Richtung</string>
+    <string name="north_west">Wind in nordwestlicher Richtung</string>
+    <string name="notif_city">Stadt: %s</string>
+    <string name="notif_temp">Temperatur: %1$d %2$s</string>
+    <string name="other_apps">ANDERE APPS</string>
+    <string name="paytm_donate">Ihre Spende hilft mir zur Entwicklung und dem Support weiterer Apps. Scannen Sie diesen QR Code mit der PayTm app auf einem anderen Telefon oder klicken Sie darauf</string>
+    <string name="pick_city">Hallo! Lassen Sie uns beginnen, indem wir eine Stadt auswählen</string>
+    <string name="place_not_found">Entschuldigung, keine Wetterdaten gefunden</string>
+    <string name="please_wait">Bitte warten</string>
+    <string name="pressure">Druck: %.2f hPa</string>
+    <string name="rain_">Regen : %.2f</string>
+    <string name="refresh">Neu laden</string>
+    <string name="reset">ZURÜCKSETZEN</string>
+    <string name="sat">Sa</string>
+    <string name="search">Suche</string>
+    <string name="snow_">Schnee : %.2f</string>
+    <string name="south">Wind aus Norden</string>
+    <string name="south_east">Wind in südöstlicher Richtung</string>
+    <string name="south_west">Wind in südwestlicher Richtung</string>
+    <string name="speed">Geschwindigkeit: %.2f</string>
+    <string name="sun">So</string>
+    <string name="sunrise">Sonnenaufgang: %s</string>
+    <string name="sunset">Sonnenuntergang: %s</string>
+    <string name="target_location_content">Tippen um Wetterdaten Ihres aktuellen Standortes abzurufen. Nach rechts streichen, um weitere Optionen zu sehen.</string>
+    <string name="target_location_title">Suche Daten für Ihren Standort</string>
+    <string name="target_refresh_content">Von oben streichen zum Neuladen</string>
+    <string name="target_refresh_title">Neu laden</string>
+    <string name="target_search_content">Suche nach Wetterdaten von über 200.000 Städten und Ortschaften</string>
+    <string name="target_search_title">Suche nach einer Stadt</string>
+    <string name="temperature">Temperatur: %1$.2f %2$s</string>
+    <string name="thu">Do</string>
+    <string name="title_activity_detail">Details</string>
+    <string name="title_activity_first_launch">Stadt eingeben</string>
+    <string name="title_activity_paytm_donate">Via PayTm spenden</string>
+    <string name="toggle_values">Werte umschalten</string>
+    <string name="tue">Di.</string>
+    <string name="uh_oh">Oh. Die Stadt, die Sie eingeben haben, existiert in der Datenbank nicht. Versuchen Sie es nochmals!</string>
```

Comment:
Yes. Thanks. I'm also struggling a bit with wind directions:
It seems as if in German the direction in which the wind blows is indicated, whereas in English the wind direction is given as the direction of the sky from which the wind blows.
Am I right?